deps/json.lua

```diff
@@ -714,17 +753,17 @@ function json.use_lpeg ()
       return obj, retpos
     end
   end
+  jsonlpeg.parse = json.decode
```

Comment:
This should be `jsonlpeg.decode` shouldn't it?

Reply:
Thanks for noticing that, would've went unnoticed forever 😅